packages/nuxt/src/vite/utils.ts

```diff
@@ -89,7 +98,7 @@ export function extractFunctionReexportQueryParameters(query: string): { wrap: s
  */
 export function constructWrappedFunctionExportQuery(
   exportedBindings: Record<string, string[]> | null,
-  entrypointWrappedFunctions: string[],
+  experimental_entrypointWrappedFunctions: string[],
```

Comment:
l: I think we do not need to use this experimental prefix here inside of the function? :)